cmake/tribits/core/package_arch/TribitsConstants.cmake

```diff
@@ -96,11 +96,11 @@ SET(${PROJECT_NAME}_PACKAGE_DEPS_FILES_DIR "cmake/dependencies")
 
 # Other stuff
 
-IF(WIN32)
+IF(${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Windows")
   #Apparently FIND_PROGRAM looks for an exact match of the file name.
   #So even though "git clone ..." is valid to use on windows we need to give the
   #full name of the command we want to run.
   SET(GIT_NAME git.cmd)
-ELSE(WIN32)
+ELSE()
   SET(GIT_NAME git)
-ENDIF(WIN32)
+ENDIF()
```

Comment:
Okay, we are just fixing the logic for modern CMake versions on Windows and this will still use 'git' for cygwin.